integration-tests/plugins/__tests__/inventory/inventory-items/index.js

```diff
@@ -278,7 +278,7 @@ describe("Inventory Items endpoints", () => {
       })
     })
 
-    it.only("Creates an inventory item using the api", async () => {
+    it.skip("Creates an inventory item using the api", async () => {
```

Comment:
todo: Remove
```suggestion
    it("Creates an inventory item using the api", async () => {
```

Reply:
this test is new and was merged using `only` 😆 , when removed it is failing.
I will check what is happening

Reply:
@pKorsholm is taking care of that in another PR. so I will keep it as it is in this one.